react/package.json

```diff
@@ -39,39 +39,37 @@
     "vtex-tachyons": "^2.3.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
     "apollo-cache-inmemory": "^1.4.3",
-    "babel": "^6.23.0",
     "babel-jest": "^24.1.0",
-    "babel-preset-env": "^1.7.0",
     "babel-preset-react-app": "^7.0.1",
-    "enzyme": "^3.8.0",
-    "enzyme-adapter-react-16": "^1.7.1",
-    "enzyme-to-json": "^3.3.5",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.1.0",
+    "jest-dom": "^3.1.2",
     "jest-transform-graphql": "^2.1.0",
-    "react-intl": "^2.8.0"
+    "react-intl": "^2.8.0",
+    "react-testing-library": "^6.0.0",
+    "regenerator-runtime": "^0.13.1"
   },
   "vtexScriptsOverride": {
     "srcPath": "."
   },
   "jest": {
     "verbose": true,
-    "modulePaths": [
+    "moduleDirectories": [
```

Comment:
It's better to keep using `modulePaths`.  `moduleDirectories` tries to find the folders recursively, so if there is two folders named `testUtils` depending how the search is done the wrong folder may be used.